pkg/pagerduty/service.go

```diff
@@ -319,12 +319,18 @@ func (c *SvcClient) CreateService(data *Data) (string, error) {
 		if newSvc == nil {
 			return "", fmt.Errorf("creating service %v returned an empty response: %w", clusterService.Name, err)
 		}
+
+		err = nil
```

Comment:
**Question:** Is there any value in resetting the error here?

Reply:
Yes, there is. `err` is eventually returned later. But yeah this is not elegant; will figure something out.